metals/src/main/scala/scala/meta/internal/metals/CodeActionProvider.scala

```diff
@@ -46,4 +50,14 @@ final class CodeActionProvider(
     Future.sequence(actions).map(_.flatten)
   }
 
+  def executeCommands(
+      codeActionCommandData: CodeActionCommandData,
+      token: CancelToken
+  )(implicit ec: ExecutionContext): Future[CodeActionCommandResult] = {
```

Comment:
```suggestion
  ): Future[CodeActionCommandResult] = {
```

nipick, this can be removed.